packages/lib/src/components/internal/FormFields/Field/Field.tsx

```diff
@@ -126,13 +126,15 @@ const Field: FunctionalComponent<FieldProps> = props => {
                         </span>
                     )}
                 </div>
+                {errorMessage && (
```

Comment:
With the new a11y guidance we want this element to _always_ be present in the DOM, regardless of whether we have an `errorMessage` or not.
This change would be a regression to how it used to be - the element only showing _if_ we have an `errorMessage` prop

Reply:
Ok, thanks, I'll revert it back.